scalameta/tokens/shared/src/main/scala/scala/meta/tokens/Token.scala

```diff
@@ -227,10 +227,14 @@ object Token {
     class Int(value: BigInt) extends NumericConstant[BigInt]
     @freeform("long constant")
     class Long(value: BigInt) extends NumericConstant[BigInt]
+    @freeform("arbitrary integer constant")
+    class IntXL(value: BigInt) extends NumericConstant[BigInt]
     @freeform("float constant")
     class Float(value: BigDecimal) extends NumericConstant[BigDecimal]
     @freeform("double constant")
     class Double(value: BigDecimal) extends NumericConstant[BigDecimal]
+    @freeform("arbitrary decimal constant")
+    class FltXL(value: AnyDecimal) extends NumericConstant[AnyDecimal]
```

Comment:
```suggestion
    class FloatXL(value: AnyDecimal) extends NumericConstant[AnyDecimal]
```
Nit, but doesn't seem worth losing the two letters to make it shorter